programs/marginfi/src/instructions/marginfi_group/add_pool.rs

```diff
@@ -22,7 +22,7 @@ use anchor_spl::token_interface::*;
 /// TODO: Allow for different oracle configurations
 pub fn lending_pool_add_bank(
     ctx: Context<LendingPoolAddBank>,
-    bank_config: BankConfig,
+    bank_config: BankConfigCompact,
 ) -> MarginfiResult {
```

Comment:
This will basically let banks exist in an invalid state, it will most likely break clients that assume that bank will never be in state where oracle setup is None